boards/cc3200-launchxl/Makefile.features

```diff
@@ -0,0 +1,7 @@
+FEATURES_PROVIDED += periph_uart
+FEATURES_PROVIDED += periph_gpio
+FEATURES_PROVIDED += periph_timer
+FEATURES_PROVIDED += periph_cpuid
```

Comment:
This is already in `cpu/cc3200/Makefile.features`

Reply:
@benpicco sorry for the late response. I am back now and will take a look at the problems